### DISCUSSION TITLE
About the closed issue about the Input Parameters

### DISCUSSION THREAD

**Discussion by SundaramMaheshkumar** [bpmnServer/bpmn-server#86] [General]
Hi @ralphhanna,

I would like to discuss about your comment on the [issue](https://github.com/ralphhanna/bpmn-server/issues/85).

>> You need to provide unique names or structure since all instance variables belong to the instance

So you mean to say, no two tasks can have the same variable name in their input parameters?
Suppose, if Task A has an input parameter named `destination`, Task B shouldn't have an input parameter named `destination`!?

Correct me if I am wrong. But this shouldn't be a problem in an ideal usecase right!
Because using the same variable names when the tasks are sequential doesn't have a problem. It executes as expected.

In the [issue](https://github.com/ralphhanna/bpmn-server/issues/85), it seems the problem occurs because they are parallel tasks.

Kindly clarify on this. Thanks!

**Comment by ralphhanna** [comment]
I agree that was a bug, now fixed
